ta/pkcs11/src/processing_symm.c

```diff
@@ -900,7 +900,7 @@ enum pkcs11_rc wrap_data_by_symm_enc(struct pkcs11_session *session,
 }
 
 enum pkcs11_rc unwrap_key_by_symm(struct pkcs11_session *session, void *data,
-				  uint32_t data_sz, void *out_buf,
+				  uint32_t data_sz, void **out_buf,
```

Comment:
These pointer-pointer changes should be in commit that originates them.

Reply:
Changed